### DISCUSSION TITLE
make first iteration to be displayed as 1

### DISCUSSION THREAD

**Discussion by joseluisulloafulgeri** [jspsych/jsPsych#1704] [Q&A] [answered]
Hi all,

I have a very basic question. I'm new in javascript and I haven't find a solution for this.
I have an experiment based on a number of blocks (nblocks). After each block I would like to tell the subject which block has done and how many are left.
I have no idea how to make the first iteration (which is 0 in javascript) to be 1.
Here's a snipet of the code I'm using.

```javascript
if(i < (nblocks-1)){ 
	test_break = {
		type: "html-keyboard-response",
		stimulus: "<p>This is block number " + i + " out of 5.</p></br>" + "<p>Please take a break.</p></br>",
		post_trial_gap: 1000,
		data: {test_part: 'break'}					
	};
```

Any hint will be appreciated!
Jose

**Comment by becky-gilbert** [comment, marked as answer]
Hi Jose, try adding 1 to i and then converting the result to a string, like this:
```js
stimulus: "<p>This is block number " + (i+1).toString() + " out of 5.</p></br>" + "<p>Please take a break.</p></br>"
```

**Reply by joseluisulloafulgeri** [reply]
This works perfect, Thanks Becky!
